src/chapters/typescript/functions.rst

```diff
@@ -0,0 +1,115 @@
+Functions in TypeScript
+=======================
+
+When creating functions in TypeScript, we have many of the same options as in JavaScript.
+We can make anonymous functions, give it a different number of parameters and so on.
+However, when working with functions in TypeScript, we have to keep *type* in mind.
```

Comment:
maybe *data types* in mind, rather than *type*

Reply:
Done! Thanks!